Samples/PublicSamples/RecordingBot/docs/tutorials/deploy/aks.md

```diff
@@ -0,0 +1,384 @@
+
+```powershell
+az login --tenant 99999999-9999-9999-9999-999999999999
+```
+
+After running the command, it should show the following message:
+
+```text
+The default web browser has been opened at https://login.microsoftonline.com/99999999-9999-9999-9999-999999999999/oauth2/v2.0/authorize. Please continue the login in the web browser. If no web browser is available or if the web browser fails to open, use device code flow with `az login --use-device-code`.
+```
+
+And a Browser window with the Microsoft Login Page should open.
+
+![Login Page](../../images/loginscreenshot.png)
+
+There we login with our Microsoft Entra Id administrator account and accept the scopes requested.
+
+After successful log in, we should see a output similar to:
+
+```json
+[
+  {
+    "cloudName": "AzureCloud",
+    "homeTenantId": "99999999-9999-9999-9999-999999999999",
+    "id": "yyyyyyyy-yyyy-yyyy-yyyy-yyyyyyyyyyyyy",
+    "isDefault": true,
+    "managedByTenants": [],
+    "name": "recordingbotsubscription",
+    "state": "Enabled",
+    "tenantId": "99999999-9999-9999-9999-999999999999",
+    "user": {
+      "name": "user@xyz.com",
+      "type": "user"
+    }
+  }
+]
+```
+
+## Create Azure Resource Group
+
+Now we can start to create resources in our azure subscription. We start with creating a resource group in our Azure Subscription.
+
+```powershell
+az group create 
+    --location westeurope 
+    --name recordingbottutorial 
+    --subscription "recordingbotsubscription"
+```
+
+The result in the command line should look something like:
+
+```json
+{
+  "id": "/subscriptions/yyyyyyyy-yyyy-yyyy-yyyy-yyyyyyyyyyyyy/resourceGroups/recordingbottutorial",
+  "location": "westeurope",
+  "managedBy": null,
+  "name": "recordingbottutorial",
+  "properties": {
+    "provisioningState": "Succeeded"
+  },
+  "tags": null,
+  "type": "Microsoft.Resources/resourceGroups"
+}
+```
+
+## Create Azure Kubernetes Cluster
+
+Now we can create an AKS cluster in the resource group, we will create a free tier cluster, this doesn't mean the nodes of the cluster are for free but the managment plane is. A free tier cluster is for testing and development and should not be used for production, see [pricing tiers](https://learn.microsoft.com/en-us/azure/aks/free-standard-pricing-tiers) for reference. And set the node count of the system nodepool, that is automatically created, to 1, because we want to avoid unnecessary cost from this tutorial. For the size of the system nodes in the system nodepool we choose the `standard_d2s_v3`-series, as this series is available at the most regions and some nodes in this series are available without requesting more quotas. Still you might need to check you're quotas, see [per vm quotas](https://learn.microsoft.com/en-us/azure/quotas/per-vm-quota-requests) for reference.

```

Comment:
```suggestion
Let's get to creating the AKS cluster in the resource group! Create a free tier cluster (This doesn't mean the nodes of the cluster are for free but the management plane is. A free tier cluster is for testing and development and should not be used for production, see [pricing tiers](https://learn.microsoft.com/en-us/azure/aks/free-standard-pricing-tiers) for reference). Set the node count of the automatically created system nodepool to 1, because we want to avoid unnecessary cost from this tutorial. Choose the `standard_d2s_v3`-series, as this series is available at most regions and some nodes in this series are available without requesting additional quotas. Still you might need to check your quotas, see [per vm quotas](https://learn.microsoft.com/en-us/azure/quotas/per-vm-quota-requests) for reference.

```

Reply:
I still believe there should be a warning block for possibly incurring costs
(its easy to overlook this detail in a block of text, especially if you're just trying to get something done quickly)